### ISSUE TITLE
probem installing latest vmware-tools chocolatey package via ansible/winrm

### ISSUE THREAD

**Issue by toxicvengeance** [bcurran3/ChocolateyPackages_CCR#37]
Hi,

We are using your chocolatey package to keep our windows servers vmware-tools up to date. Thank you for the package by the way :smiley:

We have a Jenkins job that runs every month and updates the vmware-tools package on all servers in case there is an update. 

Jenkins uses an Ansible Playbook that looks like this:

```yaml
tasks:
  - name: updating vmware-tools via chocolatey
    win_chocolatey:
      name: vmware-tools
      state: latest
```

With the newest vmware-tools package you uploaded to chocolatey I'm getting the following error when installing the package via ansible.

```powershell
2018-08-16 11:59:52,836 1784 [DEBUG] - Finishing 'Start-ChocolateyProcessAsAdmin'
2018-08-16 11:59:52,977 1784 [INFO ] - vmware-tools has been installed.
2018-08-16 11:59:53,211 1784 [ERROR] - ERROR: Exception calling "CreateToastNotifier" with "1" argument(s): "Die Benachrichtigungsplattform ist nicht verfügbar.
```

Die Benachrichtigungsplattform ist nicht verfügbar. => The notification platform is not available

This seems to come from the following line that has been added with commit 15dce7b1616e1a9800465d1576d2a7f1e734184d

```
Show-ToastMessage "$packageName installed." "Version $env:packageVersion."
```
Is there another way you can output the message ? Show-ToastMessage does not seem to work via the winrm connection ansible uses propably because it is not a graphical environment. Maybe something simple like Write-Host ?

**Comment by bcurran3** [comment]
Hi. Thanks for letting me know.
This should already be fixed in the 3.2 release:
https://chocolatey.org/packages/vmware-tools/10.3.2.9925305

***
**Click here to become a [Patron](https://www.patreon.com/bcurran3).**
***

**Comment by toxicvengeance** [comment]
Thanks. Seems to be working again
